src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+
+/**
+ * The class with deals with user interaction.
+ */
+public class Ui {
+    public static final String LINE_BREAK = "____________________________________________________________";
+
+    /**
+     * Prints a welcome message when the program starts.
+     */
+    public void welcome() {
+        System.out.println("_____________   _       _______________\n|___   __||  | | |     | |__  __| ____|"
+                + "\n    | |  | __ ||  \\   /  |  | | | |___\n _  | | | |__| |   \\_/   |  | | |  ___|"
+                + "\n| |_| | |  __  | |\\   /| |__| |_| |___\n|_____| |_|  |_|_| \\_/ |_|______|_____|");
+        System.out.println("Hi, I'm Jamie.\nWhat do you want to do?\n"
+                + "For dates and time, please enter in the format:\n"
+                + "dd/MM/yyyy HHmm eg. 29/10/2022 0000");
+    }
+
+    /**
+     * The method to list out the tasks that are present in taskList.
+     */
+    public void printTaskList(TaskList tasks) {
+        System.out.println("Here are your tasks");
+        System.out.println(LINE_BREAK);
+
+        for (int i = 1; i <= tasks.size(); i++) {
+            Task t = tasks.getTask(i - 1);
+            System.out.println(i + "." + t);
+        }
+        System.out.println(LINE_BREAK);
+    }
+
+    /**
+     * Prints out the message when a task is added.
+     * @param task The task that is added.
+     */
+    public void addTasksUi(Task task) {
+        System.out.println(LINE_BREAK);
+        System.out.println("Got it! I've added this task:\n  "
+                + task);
+    }
+
+    /**
+     * Prints out the message for when a task is being marked.
+     * @param marking The marking that is being done.
+     * @param task The task that is being marked.
+     */
+    public void markingUi(String marking, Task task) {
+        System.out.println(LINE_BREAK);
+        if (marking.equals("mark")) {
+            System.out.println("Ok! I've marked it as done.\n  "
+                    + task.toString());
+        } else {
+            System.out.println("Ok! I've marked it as undone.\n  "
+                    + task.toString());
+        }
+        System.out.println(LINE_BREAK);
+    }
+
+    /**
+     * Prints out the message for when a task is deleted.
+     * @param task The task that is deleted.
+     */
+    public void deleteUi(Task task) {
```

Comment:
Perhaps the method name could be renamed to something similar to `printDeleteTaskMessage`, as the name `deleteUi` could be misleading.